src/components/ProgressForm/ProgressHeader.tsx

```diff
@@ -1,11 +1,23 @@
+import { useGetLatestProgressQuery } from '@/app/services/progressesApi';
 import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+import { useRouter } from 'next/router';
 
 function ProgressHeader() {
+    const router = useRouter();
+    const id = router.query.id;
+    const { data, error } = useGetLatestProgressQuery(`taskId=${id}`);
+    let missedUpdates = 0;
+    if (!error) {
+        const currentDate = new Date();
+        const lastUpdateDate = new Date(data?.date as number);
+        missedUpdates = currentDate.getDate() - 1 - lastUpdateDate.getDate();
```

Comment:
How does this work?
Are we getting the difference between the 2 dates here?

Reply:
yes we are

Reply:
So if `currentDate` is 1 and `lastUpdatedDate` is 31 then the missing updates will be 1-1-31 = -31?
Correct me if I am missing something here.

And why are we subtracting -1 from it?